src/components/UserSelector.tsx

```diff
@@ -75,10 +65,10 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
               onClick={() => {
-                onChange(user);
+                dispatch(authorActions.setAuthor(user));
               }}
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here